manager/allocator/cnmallocator/manager.go

```diff
@@ -11,19 +11,18 @@ type manager struct {
 	networkType string
 }
 
-func StubManagerInit(networkType string) func(dc driverapi.DriverCallback, config map[string]interface{}) error {
-	return func(dc driverapi.DriverCallback, config map[string]interface{}) error {
+func StubManagerInit(networkType string) func(dc driverapi.DriverCallback, _ map[string]interface{}) error {
+	return func(dc driverapi.DriverCallback, _ map[string]interface{}) error {
 		return RegisterManager(dc, networkType)
 	}
 }
```

Comment:
Isn't this now dead code?

Reply:
DOH! I think it is now yes; originally I had to adjust the DriverCallback, but that's no longer true, so yup.

Reply:
- Oh; looking at it locally now; yes it's unused, but it was implemented in SwarmKit to (as I understand it) provide functionality to be used in https://github.com/moby/moby/pull/44983#discussion_r1106200444


I see that PR was closed though, but I wasn't sure if it was still needed (to be revived); do you know?

Reply:
> I see that PR was closed though, but I wasn't sure if it was still needed (to be revived); do you know?

It is not needed.

- moby/moby#44983 was carried in #3121